master/buildbot/test/unit/test_config.py

```diff
@@ -12,7 +12,10 @@
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 # Copyright Buildbot Team Members
-import __builtin__
+try:
```

Comment:
looks like future already implement a buildtins module for py2

http://python-future.org/imports.html#imports-of-builtins